src/content/reference/okteto-cli.mdx

```diff
@@ -246,8 +246,9 @@ $ okteto deploy
 | Options                                     |    Type    | Description                                                                                                                                                                                                                                                         | Default     |
 | :------------------------------------------ | :--------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :---------- |
 | _--build_                                   |   (bool)   | Force build of images in the `build` section                                                                                                                                                                                                                        | `false` |
-| <em className="no-wrap">--dependencies</em> |   (bool)   | Force deployment of repositories in the `dependencies` section                                                                                                                                                                                                      | `false` |
-| _--file_                                    |  (string)  | The location of the okteto manifest                                                                                                                                                                                                                                 | "okteto.yaml" |
+| _--dependencies_                            |   (bool)   | Force deployment of repositories in the `dependencies` section                                                                                                                                                                                                      | `false` |
```

Comment:
Is this force deployment or force destruction?

Reply:
This edit is for the `deploy` section which also needed some updates